src/mantarray_waveform_analysis/peak_detection.py

```diff
@@ -432,7 +432,6 @@ def calculate_twitch_period(
 
 def find_twitch_indices(
     peak_and_valley_indices: Tuple[NDArray[int], NDArray[int]],
-    filtered_data: NDArray[(2, Any), int],
```

Comment:
this data was only passed to this function so it could be used in the exceptions